### DISCUSSION TITLE
Exporting to CSV/XLSX etc.

### DISCUSSION THREAD

**Discussion by kovasap** [Comcast/react-data-grid#2827] [Q&A] [answered]
In the top right corner of https://adazzle.github.io/react-data-grid/#/common-features, I see buttons to export the data to csv/excel/pdf.  I originally thought those were part of this library, but I'm starting to think that is not the case.

How are those implemented?  Is there a quick and easy way to add similar buttons that can convert the data in my `react-data-grid`?

**Comment by mikereczek** [comment, marked as answer]
Check out these two files to see how they're implemented in the demo:

[https://github.com/adazzle/react-data-grid/blob/main/website/demos/exportUtils.tsx](https://github.com/adazzle/react-data-grid/blob/main/website/demos/exportUtils.tsx)
[https://github.com/adazzle/react-data-grid/blob/main/website/demos/CommonFeatures.tsx](https://github.com/adazzle/react-data-grid/blob/main/website/demos/CommonFeatures.tsx)

**Comment by noah-ellman** [comment]
You have to use the demo code and copy that code into your own codebase.
